docs/user/operator-hub.adoc

```diff
@@ -0,0 +1,271 @@
+:source-highlighter: pygments
+
+# odo integration with Operator Hub
+---
+
+When working in experimental mode, odo provides the ability to work with
+link:https://www.openshift.com/learn/topics/operators[Operators] installed on
+the cluster. It allows listing of Operators, creation of services from CRD
+(Custom Resource Definitions) provided by the Operators, printing the YAML
+definition and providing custom YAML definition to start the service from a
+CRD.
+
+[NOTE]
+====
+Installation of Operators is not a part of odo workflow. It is something that
+your OpenShift/Kubernetes cluster administrator should be able to do for you. 
+====
+
+=== Prerequisites
+
+- `odo` is installed.
+- Required Operators are installed in the project/namespace by cluster
```

Comment:
Reword this?

Perhaps something like: Operators are required to be installed in your OpenShift cluster. 

And then provide a link?